### ISSUE TITLE
Animations is no longer with sync with refresh rate of the monitor

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#137]
After https://android-review.googlesource.com/c/platform/frameworks/support/+/1449160 animations on desktop is no longer with sync with the current refresh rate of the monitor.

Snippet to check:
```
import androidx.compose.animation.animate
import androidx.compose.animation.core.TweenSpec
import androidx.compose.desktop.Window
import androidx.compose.runtime.mutableStateOf

fun main() {
    val target = mutableStateOf(100f)
    Window {
        val value = animate(
            target.value,
            animSpec = TweenSpec(durationMillis = 2000)
        )
        target.value = 0f
        println("$value ${System.currentTimeMillis()}")
    }
}
```

To sync animations we should not use Recomposer.current().

After https://android-review.googlesource.com/c/platform/frameworks/support/+/1478821 we can now use a local Recomposer and MonotonicFrameClock inside each Window.

**Comment by igordmn** [comment]
Partial temporary fix:
https://android-review.googlesource.com/c/platform/frameworks/support/+/1507761

**Comment by igordmn** [comment]
Should be fixed in 0.3.0-build146

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5596">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
